app/models.py

```diff
@@ -19,38 +18,59 @@
 )
 
 LOCATION_CHOICES = (
-    ("IN-HOME", "IN-HOME"),
-    ("IN-LAB", "IN-LAB"),
+    ('IN-HOME', 'IN-HOME'),
+    ('IN-LAB', 'IN-LAB'),
+)
+
+USER_ROLE_CHOICES = (
+    ('ADMIN', 'admin'),
+    ('PATIENT', 'Patient'),
+    ('TECHNICIAN', 'Technician')
+)
+
+REQUEST_STATUS_CHOICES = (
+    ('APROVED', 'Aproved'),
+    ('DECLINED', 'Declined'),
+    ('PENDING', 'Pending')
 )
 
 
 class Address(models.Model):
-    address_line = models.CharField(max_length=200)
+    first_line = models.CharField(max_length=200)
+    second_line = models.CharField(max_length=200)
     city = models.CharField(max_length=200)
-    zipcode = models.IntegerField()
-    country = models.CharField(max_length=20, default='Puerto Rico')
+    zip_code = models.CharField(max_length=15)
+    country = models.CharField(max_length=20, default='Puerto Rico')   
```

Comment:
I think we should leave the model without a state variable because we only plan to offer the service in the PR

Reply:
ok